public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -767,8 +771,10 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> impleme
   getSaveResource(options: SaveDashboardAsOptions): ResourceForCreate<unknown> {
```

Comment:
Should we move this to the serializer? So each version of the serializer generates this payload correctly

Reply:
yeah, we should, glad you have noticed that

Reply:
no